include/SDL3/SDL_hints.h

```diff
@@ -1737,7 +1737,19 @@ extern "C" {
 #define SDL_HINT_RENDER_DIRECT3D11_DEBUG    "SDL_RENDER_DIRECT3D11_DEBUG"
 
 /**
- * A variable specifying which render driver to use.
+ *  A variable controlling whether to enable Vulkan Validation Layers
+ *
+ *
+ *  This variable can be set to the following values:
+ *    "0"       - Disable Validation Layer use
+ *    "1"       - Enable Validation Layer use
+ *
+ *  By default, SDL does not use Vulkan Validation Layers.
+ */
+#define SDL_HINT_RENDER_VULKAN_DEBUG    "SDL_RENDER_VULKAN_DEBUG"
+
+/**
+ *  A variable controlling whether the Direct3D device is initialized for thread-safe operations.
```

Comment:
```suggestion
 * A variable specifying which render driver to use.
```

Reply:
Oh, yes, this was a merge error, thanks.